src/commands/functions/create.js

```diff
@@ -1,9 +1,10 @@
-const fs = require('fs-extra')
+const fs = require('fs')
```

Comment:
This file is only using `*Sync`/`createWriteStream` methods, so no need for `util.promisify`.

However, the code is calling `mkdir` with the `recursive` flag which requires Node.js >= v10.12.0.

What would be the best way to handle this?